.github/workflows/skip-for-docs.yml

```diff
@@ -0,0 +1,33 @@
+# A way to merge documentation changes without running the tests
+name: On Docs Change (PR)
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths:
+      - "docs/**"
+
+jobs:
+  static-analysis:
+    name: Static Analysis (linting, vulns)
```

Comment:
eventually we could have real formatting for the docs here

Reply:
Yep! We will also have a real build step in the end, because we want the thing to end up on the academy